packages/teams/src/utils/filter-user-privs-by-subscription-type.ts

```diff
@@ -0,0 +1,40 @@
+import { IUser } from "@esri/arcgis-rest-auth";
+import { cloneObject, includes, without } from "@esri/hub-common";
+
+const ALLOWED_SUBSCRIPTION_TYPES = [
+  "Demo & Marketing",
+  "Organizational Plan",
+  "Community",
+  "In House",
+  "ConnectED",
+  "ELA",
+  "Education Site License",
+  "Education",
+  "HUP Online"
+];
+
+/**
+ * Returns a cloned copy of the user object with updated privileges
+ * based on whether or not the user has a subscription type not in the
+ * allowed list
+ * @param {object} user
```

Comment:
Add a note that if/when AGO implements this logic on their side, we can remove this function or simply return the passed in user